cli/account_command.go

```diff
@@ -30,7 +30,7 @@ import (
 	"github.com/nats-io/jsm.go/api"
 	"github.com/nats-io/nats-server/v2/server"
 	"github.com/nats-io/nats.go"
-	"github.com/nats-io/natscli/columns"
+	"github.com/mprimi/natscli/columns"
```

Comment:
All of these are in a single commit tagged with message `[TEMP / DO NOT MERGE] Rename project to avoid name clash` so yes, will definitely not include in the final/real PR.